app/javascript/custom/page.js

```diff
@@ -0,0 +1,10 @@
+const href = location.href;
+// ヘッダーの中のaタグを全部取得
+var links = document.querySelectorAll(".header > .navbar-nav > .nav-item > .nav-link");
+
+// ループでURLと一致したaタグに current を付ける。
+for (var i = 0; i < links.length; i++) {
```

Comment:
使えるのであれば`const` `let`のほうがよさそう

```suggestion
const links = document.querySelectorAll(".header > .navbar-nav > .nav-item > .nav-link");

// ループでURLと一致したaタグに current を付ける。
for (let i = 0; i < links.length; i++) {
```

Reply:
こちらは別の方法で実装できたのでpage.jsごと削除します